libnetwork/portmapper/mapper_linux_test.go

```diff
@@ -195,6 +196,8 @@ func TestMapAllPortsSingleInterface(t *testing.T) {
 }
 
 func TestMapTCPDummyListen(t *testing.T) {
+	testutil.SkipWhenUnprivileged(t)
```

Comment:
If we change the ports in this test to unprivileged ones it will at least work (assuming the ports aren't taken...) with non-root.